pages/[profileAddress]/send.vue

```diff
@@ -61,11 +61,15 @@ const handleSend = async () => {
       const transaction = {
         from: connectedProfile.value?.address,
         to: receiver.value?.address as unknown as string,
-        value: toWei(sendAmount.value || '0', 'ether'),
-        gasLimit: GAS_LIMIT,
+        value: numberToHex(toWei(sendAmount.value || '0', 'ether')),
+        // gasLimit: numberToHex(GAS_LIMIT),
```

Comment:
maybe remove this?

Reply:
Since this is temporary fix I wanted to keep this (it will be needed when we go back to web3 solution)